gmso/core/topology.py

```diff
@@ -610,6 +611,34 @@ def update_topology(self):
         self.update_connection_types()
         self.is_typed(updated=True)
 
+    def get_index(self, member):
+        """Get index of a member in the topology
+
+        Parameters
+        ----------
+        member : one of {gmso.Site, gmso.Bond, gmso.Angle, gmso.Dihedral, gmso.Improper, gmso.Connection}
```

Comment:
I think remove `gmso.Connection` since we agreed today that initializing a generic `Connection` would not be valid.

Reply:
Could we also change this to:
```
member : A GMSO core object of `Site`, `Bond`, `Angle`, `Dihedral`, or `Improper`    
```

Also could we add a `Notes` section? Something like:
```
Notes
------
`get_index` method currently not supported for Potential objects.
```